File: gb-angular-frontend/src/app/interfaces/package-item.ts
import {ProductItem} from './product-item';

export interface PackageItem {
  package_item_id?: number;
  product_item_id: number;
  item_name: string;
  package_grp_id: number;
  package_id: number;
  status: string;
  popularity: number;
  likes: number;
  package_item_image: string
  productItem: ProductItem;
}

File: gb-spring-backend/src/main/java/com/egroupbuy/model/PackageItem.java
package com.egroupbuy.model;

import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class PackageItem {
    int package_item_id;
    int product_item_id;
    String item_name;
    int package_grp_id;
    int package_id;
    String status;
    int popularity;
    int likes;
    String package_item_image;
    ProductItem productItem;
}
